test/functional/api/command_spec.lua

```diff
@@ -160,7 +154,6 @@ describe('nvim_get_commands', function()
       definition = 'call \128\253R4_just_great()',
       name = 'Cmd4',
       nargs = '0',
-      preview = false,
```

Comment:
I was going to ask what is the behavior when called via RPC. It looks like the API automatically omits the Lua function results, correct?

Actually, all of these cases were `preview=false`, so maybe this test needs a tweak to confirm what happens in the "truthy" case. (I.e. define a `preview` function, and perhaps also a string command if that is not already tested.)